lib/src/cli.rs

```diff
@@ -699,6 +708,19 @@ async fn container_import(
     Ok(())
 }
 
+/// Grouping of metadata about an object.
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct RawMeta {
+    /// When the image was created. Sync it with the io.container.image.created label.
+    pub created: Option<String>,
+    /// Top level labels, to be prefixed to the ones with --label
+    pub labels: Option<BTreeMap<String, String>>,
+    /// ContentId to layer annotation
+    pub layers: IndexMap<String, String>,
+    /// OSTree hash to layer ContentId
+    pub mapping: IndexMap<String, String>,
```

Comment:
I think we could really use an example of this, a bit more information. Also one thing we lack for this project in general is man pages for the CLI (this is my fault)...most of the current CLI arguments I think are somewhat self-explanatory, but this one isn't.

Reply:
Indeed. Need to figure out proper names for them as well.